protocol/testing/e2e/gov/add_new_market_test.go

```diff
@@ -340,7 +369,32 @@ func TestAddNewMarketProposal(t *testing.T) {
 
 				// Check that clob pair is updated.
 				require.Equal(t, msgUpdateClobPairToActive.ClobPair, clobPair)
-				// TODO(CORE-585): Check that orders cannot be placed if no valid oracle price update has occurred.
+
+				// Advance to some blocks after, and place an order on the market.
+				ctx = tApp.AdvanceToBlock(uint32(ctx.BlockHeight())+NumBlocksAfterTradingEnabled, testapp.AdvanceToBlockOptions{})
+				price, err := tApp.App.PricesKeeper.GetMarketPrice(ctx, testMarketParam.Param.Id)
+				require.NoError(t, err)
+				// No oracle price updates were made.
+				require.Equal(t, uint64(0), price.Price)
```

Comment:
Suggestion: can you use [`require.Zero`](https://pkg.go.dev/github.com/stretchr/testify/require#Zero)?